idle_sweep_test.go

```diff
@@ -97,6 +131,15 @@ func connectionStatus(channels []*Channel) string {
 	return strings.Join(status, ", ")
 }
 
+func exchangeStatus(channels []*Channel) string {
```

Comment:
optional: rather than a separate method, may be simpler to have `allExchangeEmpty` return an error string if there's non-empty ones, and merge this logic in there.

Reply:
done